content/inferno/time-series/time-series.md

```diff
@@ -0,0 +1,309 @@
+
+- [ROCKET: Exceptionally fast and accurate time series classification using random convolutional kernels](https://arxiv.org/abs/1910.13051)
+
+Another extremely effective method that is always worth trying is that of the [matrix profile](#The-Matrix-Profile) (next section).
+
+::: tip
+Some methods are more precise, others more computationally performing, others can find complex relationships in the data (neural networks). 
+**A fundamental point** is that _rarely_ it is worth choosing complex methods (such as a complicated recurrent neural network) over more "simple" ones, such as BOSS or DTW.
+:::
+
+Equipped with the methods considered here, it's almost guaranteed that you will find the perfect fit for the requirements of your problems.
+
+## The Matrix Profile
+
+From the [The UCR Matrix Profile Page](https://www.cs.ucr.edu/~eamonn/MatrixProfile.html) website:
+
+The Matrix Profile (and the algorithms to compute it: STAMP, STAMPI, STOMP, SCRIMP, SCRIMP++, SWAMP, and GPU-STOMP), has the potential to **revolutionize time series data mining** because of its **generality, versatility, simplicity, and scalability**.  In particular, it has implications for:
+
+- time series motif discovery
+- time series joins
+- shapelet discovery (classification)
+- density estimation
+- semantic segmentation
+- visualization
+- rule discovery
+- clustering 
+
+To learn how to use the Matrix Profile for your times series classification problems go through these _invaluable_ resources:
+
+- [Time Series Data Mining Using the Matrix Profile: A Complete Tutorial](https://www.cs.ucr.edu/~eamonn/Matrix_Profile_Tutorial_Part1.pdf)
+- [100 Time Series Data Mining Questions and Answers](https://www.cs.ucr.edu/~eamonn/100_Time_Series_Data_Mining_Questions__with_Answers.pdf)
+
+The recommended Python package that Virgilio suggests you to work with the Matrix Profile is [**Stumpy**](https://seanlaw.github.io/2019/05/13/stumpy/) (docs [here](https://stumpy.readthedocs.io/en/latest/Tutorial_STUMPY_Basics.html)), which implements the latest and most efficient methods to calculate the Matrix Profile for your time series. 
+
+> The Author of the package ([Sean Law](@seanmylaw)) contributed to this guide too, reviewing it on GitHub and helping Virgilio collecting all the resources. You can find a lot of detailed tutorials about using Stumpy for the matrix profile on his [Medium Page](https://medium.com/@seanmylaw/stumpy-fdb9f8f1f261) or get in touch with him at his [LinkedIn Page](https://www.linkedin.com/in/seanlawphd/). In addition, he's very active in answering questions and giving tips about Tumpy in the ["Issues" section](https://github.com/TDAmeritrade/stumpy/issues?q=) of the GitHub project!
+
+One of the (very many) advantages of the Matrix Profile is that you can calculate it _to feed it_ to a more traditional method of supervised classification. 
+
+In this sense, the Matrix Profile is also a method of automatic features extraction!
+
+Other useful links are:
```

Comment:
Also, recommend adding this resource on Time-Series https://github.com/neomatrix369/awesome-ai-ml-dl/tree/master/time-series_anomaly-detection#time-series--anomaly-detection to the list, many of the links mentioned in this guide and more are present there